packages/progress-indicator/test/lion-progress-indicator.test.js

```diff
@@ -1,18 +1,65 @@
-import { expect, fixture } from '@open-wc/testing';
 import { html } from '@lion/core';
+import { expect, fixture as _fixture } from '@open-wc/testing';
 
 import '@lion/progress-indicator/define';
 
+/**
+ * @typedef {import('../src/LionProgressIndicator').LionProgressIndicator} LionProgressIndicator
+ * @typedef {import('@lion/core').TemplateResult} TemplateResult
+ */
+
+const fixture = /** @type {(arg: TemplateResult) => Promise<LionProgressIndicator>} */ (_fixture);
+
 describe('lion-progress-indicator', () => {
-  describe('Accessibility', () => {
-    it('adds a label', async () => {
+  describe('indeterminate', () => {
+    it('by default', async () => {
       const el = await fixture(html` <lion-progress-indicator></lion-progress-indicator> `);
+      expect(el.getAttribute('role')).to.equal('status');
+    });
+
+    it('adds a label by default', async () => {
+      const el = await fixture(html` <lion-progress-indicator></lion-progress-indicator> `);
+      await el.localizeNamespacesLoaded;
       expect(el.getAttribute('aria-label')).to.equal('Loading');
     });
+  });
+
+  describe('determinate', async () => {
+    it('once value is set', async () => {
+      const el = await fixture(
+        html` <lion-progress-indicator value="25"></lion-progress-indicator> `,
+      );
+      expect(el.getAttribute('role')).to.equal('progressbar');
+      expect(el.getAttribute('aria-valuenow')).to.be.eq('25');
+    });
 
-    it('sets the right role', async () => {
+    it('allows to set min & max values', async () => {
+      const el = await fixture(
+        html` <lion-progress-indicator value="25"></lion-progress-indicator> `,
+      );
+      expect(el.getAttribute('aria-valuemin')).to.be.eq('0');
+      expect(el.getAttribute('aria-valuemax')).to.be.eq('100');
```

Comment:
all aria attrs can go inside an a11y describe

Reply:
I would actually expect tests here that allow to define min and max. Also after init